Bottom: 4f107a7259394aab872708271a086ea950cc262e
Top:    bb6a4a91788f313bb6ced2847544624141348af8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 17:55:13 +0100

Refresh of misc-c-add-query-function-ispa

---

diff --git a/src/misc.c b/src/misc.c
index d7aad5d..d63aef1 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -600,7 +600,7 @@ IsLayerGroupEmpty (int num)
   return true;
 }
 
-static bool
+bool
 IsPasteEmpty (int side)
 {
   bool paste_empty = true;
